<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QWidget" name="Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>346</height>
   </rect>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabwidget">
     <widget class="QWidget" name="tab_general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_nickname">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>60</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="TextLabel1_2_2_2_2">
         <property name="text">
          <string>E-Mail:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="ActionLineEdit" name="le_fullname"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="TextLabel1_3_2_3_2">
         <property name="text">
          <string>Birthday:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" rowspan="5">
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="tb_photo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>129</width>
             <height>129</height>
            </size>
           </property>
           <property name="toolTip">
            <string>View in real size</string>
           </property>
           <property name="shortcut">
            <string>Alt+V</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pb_open">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&amp;Open...</string>
             </property>
             <property name="shortcut">
              <string>Alt+O</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_clear">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&amp;Clear</string>
             </property>
             <property name="shortcut">
              <string>Alt+C</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="TextLabel1_2_2">
         <property name="text">
          <string>Full Name:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="TextLabel1_3_2_2_2_2">
         <property name="text">
          <string>Phone #:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="TextLabel1_2_3_2">
         <property name="text">
          <string>Homepage:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="TextLabel1_2">
         <property name="text">
          <string>Nickname:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="le_phone"/>
       </item>
       <item row="2" column="1">
        <widget class="ActionLineEdit" name="le_bday">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="ActionLineEdit" name="le_homepage"/>
       </item>
       <item row="5" column="1">
        <widget class="ActionLineEdit" name="le_email"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_work">
      <attribute name="title">
       <string>Work</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <layout class="QGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_title"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="TextLabel1_2_2_2_3">
           <property name="text">
            <string>Company:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="TextLabel1_3_2_2_2_3_2">
           <property name="text">
            <string>Position:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_orgName"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TextLabel1_2_3_3">
           <property name="text">
            <string>Department:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_orgUnit"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="le_role"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="TextLabel1_3_2_2_2_4">
           <property name="text">
            <string>Role:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_location">
      <attribute name="title">
       <string>Location</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <layout class="QGridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="5" column="0">
          <widget class="QLabel" name="TextLabel1_3_2_2_2">
           <property name="text">
            <string>Country:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="TextLabel1_3_2_2_2_3">
           <property name="text">
            <string>Postal Code:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_ext"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="le_country"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="le_pcode"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="TextLabel1_2_3">
           <property name="text">
            <string>State:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="TextLabel1_2_2_2">
           <property name="text">
            <string>City:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_street"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="TextLabel1_3_2_3">
           <property name="text">
            <string>Street:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="le_state"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="le_city"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_about">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="te_desc"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_status">
      <attribute name="title">
       <string>Status</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="PsiTextView" name="te_status" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ActionLineEdit</class>
   <extends>QLineEdit</extends>
   <header>actionlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>PsiTextView</class>
   <extends>QWidget</extends>
   <header>psitextview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabwidget</tabstop>
  <tabstop>le_fullname</tabstop>
  <tabstop>le_nickname</tabstop>
  <tabstop>le_bday</tabstop>
  <tabstop>le_phone</tabstop>
  <tabstop>le_homepage</tabstop>
  <tabstop>le_email</tabstop>
  <tabstop>tb_photo</tabstop>
  <tabstop>pb_open</tabstop>
  <tabstop>pb_clear</tabstop>
  <tabstop>le_orgName</tabstop>
  <tabstop>le_orgUnit</tabstop>
  <tabstop>le_title</tabstop>
  <tabstop>le_role</tabstop>
  <tabstop>le_street</tabstop>
  <tabstop>le_ext</tabstop>
  <tabstop>le_city</tabstop>
  <tabstop>le_state</tabstop>
  <tabstop>le_pcode</tabstop>
  <tabstop>le_country</tabstop>
  <tabstop>te_desc</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
